addon/mixins/modal.js

```diff
@@ -25,25 +25,41 @@ export default Ember.Mixin.create(Base, {
 
   onHide: function() {
     this.set('hiding', true);
+  },
+
+  onHidden: function() {
     if (this.get('controller')) {
```

Comment:
Yes, it was around on mine. The reason it is, is we are changing the workflow. Instead of calling closeModal, you should override approve or cancel to stop it from closing. Order of events should be
With controller overrides
```
template click on a semantic action 
-> modal mixin 
-> controller actions 
-> controller finishes action 
-> controller calls view.execute('hide') 
-> semantic calls onHidden
-> mixin call closeModal
```
without controller overrides
```
template click on a semantic action 
-> modal mixin 
-> semantic calls onHidden 
-> mixin calls closeModal
```